src/mixxx.cpp

```diff
@@ -449,6 +449,11 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
             m_pLibrary,
             &Library::onSkinLoadFinished);
 
+    connect(this,
+            &MixxxMainWindow::newSkinLoaded,
+            WaveformWidgetFactory::instance(),
+            &WaveformWidgetFactory::onSkinLoadFinished);
```

Comment:
We currently have two different naming schemes in our codebase: `onSomeSignal` and `slotSomeSignal`. We should agree on a single way of naming things. I think the latter is more prevalent atm but the former is nicer on the eyes IMHO. Opinions?

Reply:
I don't have a strong opinion either way, but it's easier to go with `slotSomeSignal` because it is already used more.

Reply:
As I say below, the newSkinLoaded signal already existed and I duplicated the connect that existed for Library, even the naming. 
But indeed, making it explicit that this is expected to be called from a signal is better.
So, what do I do?